### PR TITLE
Agregar MCCcodes

### PR DESCRIPTION
**Pull request by NicolasFernandezGo**
## Description
In few word, describe what’s your Pull Request about.

### Issue involved
Fixes #<issue>

### Checklist:
Before sending your contribution, please specify the following: 

- [x] This request modifies code snippets. 
- [x] The contribution aligns with the information stated in the repository wiki.

- [ ] Contribution incorporates an article not included until now, which must be added to indexes as a new section. 
- [ ] Contribution includes new features. 
- [ ] New features were communicated in changelog.
- [ ] Requires translations.

### CODE REVIEW

File: reference/pos/resource.yaml [hunk 1]
```diff
@@ -31,9 +31,38 @@ properties:
 - category:
     type: Long
     description:
-      en: MCC code that indicates the category of the Point of Sale.
-      es: Código MCC que indica el rubro del Punto de Venta.
-      pt: Código MCC que indica o categoria do ponto de venda.
+      en: MCC code that indicates the category of the Point of Sale. If not specified, it remains as a generic category
+      es: Código MCC que indica el rubro del Punto de Venta. Si no se especifica, queda como categoria generica.
+      pt: Código MCC que indica o categoria do ponto de venda. Se não especificado, ele permanece como uma categoria genérica
```
Comment (adfonseca) [inline reference/pos/resource.yaml]: Código MCC que indica **a** categoria do ponto de venda. Se não especificado, ele permanece como uma categoria genérica.

File: reference/pos/resource.yaml [hunk 2]
```diff
@@ -31,9 +31,38 @@ properties:
...
+    enum:
+    - Argentina:
+        en: Gastronomy - 621102 <br/> Gas station - 473000
+        es: Gastronomía - 621102 <br/> Petrolera - 473000
+        pt: Gastronomia - 621102 <br/> Estação de serviço - 473000
```
Comment (adfonseca) [inline reference/pos/resource.yaml]: pt: Gastronomia - 621102 <br/> Posto de combustível - 473000

File: reference/pos/resource.yaml (hunk 1 above)
Comment (NicolasFernandezGo) [reply reference/pos/resource.yaml]: Listo

File: reference/pos/resource.yaml (hunk 2 above)
Comment (NicolasFernandezGo) [reply reference/pos/resource.yaml]: Listo

**Pull request merged**